deps/thpool/thpool.c

```diff
@@ -79,18 +102,44 @@ typedef struct thread {
   struct redisearch_thpool_t* thpool_p; /* access to thpool          */
 } thread;
 
+/* On crash backtrace report */
+typedef enum {
+  FINE,
+  CRASHED,
+} statusOnCrash;
+struct thread_bt_data{
+  int trace_size;                   /* number of address in the backtrace */
+  statusOnCrash status_on_crash;    /* thead's status when the crash happened */
+  char **printable_bt;              /* back trace symbols */
```

Comment:
Couldn't fund any guarantees in the backtrace_symbols() man

Reply:
If we cannot rely on anything here, could characters such as \n be sent in a way that could damage the protocol message?
If so, we should make sure we avoid it.

Reply:
It doesn't seem reasonable to me that a system call would add special characters such as \n to the string without mentioning it
`The symbolic representation of each address consists of the function
 name (if this can be determined), a hexadecimal offset into the
function, and the actual return address (in hexadecimal). `